RecoParticleFlow/PFProducer/src/PFEGammaAlgo.cc

```diff
@@ -752,11 +714,11 @@ EvaluateSingleLegMVA(const pfEGHelpers::HeavyObjectCache* hoc,
   //use this to store linkdata in the associatedElements function below  
   const PFBlock::LinkData& linkData =  block.linkData();  
   //calculate MVA Variables  
-  chi2=elements[track_index].trackRef()->chi2()/elements[track_index].trackRef()->ndof(); 
-  nlost=elements[track_index].trackRef()->hitPattern().numberOfLostHits(HitPattern::MISSING_INNER_HITS); 
-  nlayers=elements[track_index].trackRef()->hitPattern().trackerLayersWithMeasurement(); 
-  track_pt=elements[track_index].trackRef()->pt();  
-  STIP=elements[track_index].trackRefPF()->STIP();  
+  const float chi2     = elements[track_index].trackRef()->chi2()/elements[track_index].trackRef()->ndof(); 
+  const float nlost    = elements[track_index].trackRef()->hitPattern().numberOfLostHits(HitPattern::MISSING_INNER_HITS); 
+  const float nlayers  = elements[track_index].trackRef()->hitPattern().trackerLayersWithMeasurement(); 
+  const float track_pt = elements[track_index].trackRef()->pt();  
+  const float STIP     = elements[track_index].trackRefPF()->STIP();  
```

Comment:
since these parts are edited anyways, it would be nice to use lower case to better conform to the CMSSW style